README.md

```diff
@@ -67,11 +67,11 @@ This package does not export Web Components registered as [custom elements](http
 `fast-components` is a library of Web Components that *composes* the exports of `fast-foundation` with stylesheets aligning to the FAST design language. This composition step registers a custom element. See the [quick start](http://fast.design/components/getting-started) to get started using the components.
 
 ### fast-components-msft
-
+**This package has been moved to [FluentUI](https://github.com/microsoft/fluentui/)**
 [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 [![npm version](https://badge.fury.io/js/%40microsoft%2Ffast-components-msft.svg)](https://badge.fury.io/js/%40microsoft%2Ffast-components-msft)
```

Comment:
Just checking, we're keeping the existing package name for a little while longer, correct?